de_templates/ingest/Ingest_SourceType_SourceName_DQ/config/data_quality/ingest_dq.json

```diff
@@ -0,0 +1,23 @@
+{
+    "gx_directory_path": "/dbfs/great_expectations/",
+    "dataset_name": "SalesLT",
+    "datasource_config": [
+        {
+            "datasource_name": "SalesLT.Product",
+            "datasource_type": "parquet",
+            "data_location": "abfss://raw@{ADLS_ACCOUNT}.dfs.core.windows.net/example_azuresql_1/SalesLT.Product/v1/*/*/*",
```

Comment:
Like with the other config file, would be good to make reflect the same generic examples.

Reply:
done